### ISSUE TITLE
No longer works 

### ISSUE THREAD

**Issue by Myrkie** [zynerd/RevertSteam#5]
Steam now reboots into the new library ui regardless of using this tool or the separate files 

Run program and patch steam then boots into the new library ui before applying patch

**Comment by molitar** [comment]
Can someone please fix this? The old files seem to be floating around so can someone please fix this I am unable to play any of my games anymore since a steam crashed and it had to be reinstalled. :(

If someone has old working UI library can probably use a software like Beyond Compare and archive the old files than just extract over the new.

**Comment by Myrkie** [comment]
I totally forgot I used this tool at all and reinstalled when I was in a rush to play

**Comment by Myrkie** [comment]
seems to package folder is no longer even used by steam anymore

**Comment by molitar** [comment]
Actually it still is as I see it while updating updating with new file names.. the 2 xml files does not seem to be used anymore to tell it what to install.

So if someone still has the old working UI library than they could archive all but the sub folders config, steam, and steamapps.  I believe we maybe able to use Beyond Compare to determine what files were changed and overwrite them with the archived version to get the Old UI Library back.  But need someone that still has the old Steam Library UI.

Know anybody that has it still that has not reinstalled?

**Comment by Myrkie** [comment]
seems to UI is now served from the ClientUi folder now removing that folder bricks the entire library UI  so yes if we could get these files wouldn't be to hard to update this tool to download the new ones from a new page and overwrite

going to check with some friends to see if anyone else has the old UI

**Comment by molitar** [comment]
Hopefully some of your friends has it.. <crosses fingers>

**Comment by Myrkie** [comment]
agreed and hopefully they didn't completely patch this as when I reinstalled it was because steam was failing to download file updates and was in a rush to play so hopefully that update was just a fluke

**Comment by molitar** [comment]
well it still worked for me as I had made the package folder read only.. so it could not update.. it was not until my steam crashed had an error that I had to reinstall :(

**Comment by molitar** [comment]
> 
> 
> agreed and hopefully they didn't completely patch this as when I reinstalled it was because steam was failing to download file updates and was in a rush to play so hopefully that update was just a fluke

Any luck on finding a friend that has the original version?  If I had the archive I can use Beyond Compare and find out what files changed.  I expect it's some dll or sys files that has the code to block being able to run with old UI.  I also expect they added new files and them will have to be removed. If we can only get an archive to work from where it still worked.

**Comment by Myrkie** [comment]
Havent had any lucky finding friends still on the old version sadly if I had the files wouldnt be hard to edit Revert Steam to just overwrite files since its wrote in Csharp

**Comment by luclu7** [comment]
TL;DR, but I still have my "old" installation, but you can't start game like CS:GO anymore, it asks you to restart Steam to update. :(

**Comment by Myrkie** [comment]
can you send over its files for the main install directory if you do obviously dont include the steamapps folder and dont include to two files that start with `ssfn` and userdata
the ssfn files are steam security files and store your steam guard and machine identification with these files anyone could authenticated into your account without requiring that forum of identification example files being 

`ssfn38942997300211794`
`ssfn6969601904129387765`

**Comment by luclu7** [comment]
I zipped it and uploaded to [luclu7.fr/pub/Steam.pub.zip](https://luclu7.fr/pub/Steam.pub.zip). I hope it's what you need.

**Comment by Myrkie** [comment]
seem to be having issues getting this to properly start ill have to figure this out later today when im more free thanks for providing the files

**Comment by Myrkie** [comment]
not having any luck so im starting to think its either a registry change aswell or steam servers is now blocking this version from authicating as it hangs on loading steamprofile

**Comment by molitar** [comment]
Good chance it's a single dll or sys file that changed for the connection.  Try Beyond Compare trial by Scooter Software to compare

**Comment by Myrkie** [comment]
thats my guess and yeah I tried using Beyond but ive never used it before so I dont exactly understand its output my guess is that its the steamclientUI.dll but steam refuses to start inless the dll is paired properly with the DLL

**Comment by Myrkie** [comment]
went to reddit for some insight and seems that on june 8th 2 days after I created this thread and when my steam stopped working steam has changed something internally that doesn't allow for the old UI to function anymore requiring an update [R/Steam Library UI](https://www.reddit.com/r/Steam/comments/hbjze6/help_old_library_ui/)

ill leave this thread open for any who wish to add or find a solution but I dont see one happening

**Comment by Myrkie** [comment]
so the thread user neyodaz made a point of using this launch command with a shortcut for minimalist steam 
`"C:\Program Files (x86)\Steam\Steam.exe" -console -nofriendsui -no-browser +open "steam://open/minigameslist"`

and even brings back the old chat but using the steam page is now completly impossible because of the -no-browser parameter

**Comment by Jonius7** [comment]
> not having any luck so im starting to think its either a registry change aswell or steam servers is now blocking this version from authicating as it hangs on loading steamprofile

Yeah I believe the main reason is the package format change has been implemented in the client. 
https://github.com/SteamDatabase/SteamAppInfo

It might also be this: https://twitter.com/SteamDB/status/1268252532746395654?s=20

For me, on June 18, I could log in but the client with the old UI would never show up. So I am now customising the CSS to make the new UI less terrible

**Comment by molitar** [comment]
> 
> 
> > not having any luck so im starting to think its either a registry change aswell or steam servers is now blocking this version from authicating as it hangs on loading steamprofile
> 
> Yeah I believe the main reason is the package format change has been implemented in the client.
> https://github.com/SteamDatabase/SteamAppInfo
> 
> It might also be this: https://twitter.com/SteamDB/status/1268252532746395654?s=20
> 
> For me, on June 18, I could log in but the client would never show up. So I am now customising the CSS to make the new UI less terrible

Are you able to get back the layout with small horizontal grid or at least horizontal layout and not the photo layout that makes the images clash so bad that I get a headache to look at it.  Also anything you can do about the Whats new?  Make it collapsible or make it so tiny you can't see it as it appears nobody has found a way to hide it completely anymore.

**Comment by molitar** [comment]
Ok the progress I was able to determine that you can get it to load without doing update checks and the only file that matters is this one.

\Program Files (x86)\Steam\bin\cef\cef.win7x64\v8_context_snapshot.bin

This single file is responsible for the entire layout of the browser.

**Comment by Myrkie** [comment]
yeah CEF is the browser type Cerf Sharp browser is chromium based and is an alternative to electron based app development with a browser backend

**Comment by Jonius7** [comment]
> > > not having any luck so im starting to think its either a registry change aswell or steam servers is now blocking this version from authicating as it hangs on loading steamprofile
> > 
> > 
> > Yeah I believe the main reason is the package format change has been implemented in the client.
> > https://github.com/SteamDatabase/SteamAppInfo
> > It might also be this: https://twitter.com/SteamDB/status/1268252532746395654?s=20
> > For me, on June 18, I could log in but the client would never show up. So I am now customising the CSS to make the new UI less terrible
> 
> Are you able to get back the layout with small horizontal grid or at least horizontal layout and not the photo layout that makes the images clash so bad that I get a headache to look at it. Also anything you can do about the Whats new? Make it collapsible or make it so tiny you can't see it as it appears nobody has found a way to hide it completely anymore.

Discussion here of progress: https://steamcommunity.com/discussions/forum/0/2451595019863406679/

I've shrunk the game list on the left sidebar to show more games, and on the game page swapped the columns around, so, Friends played, Achievements, Screenshots, gets a lot more space.

Removal of the What's New is fairly simple using CSS, just set it to display:none. To have it collapsible would need JavaScript, I am running into limitations of CSS modification only. Unless someone knows how to inject JavaScript on client page load or edit minified JS, I am limited to just editing CSS for now.

I mean, you cou manually open the Chrome Dev tools and inject JS via the console that way.

Trying to bring back the old Grid and List view would likely require JS.

**Comment by molitar** [comment]
I just read this on a Steam thread posted on 6/29

`Guys, I think the old client can still live on a while longer...

I pasted in my old packageinfo.vdf file into the appcache folder and the old client works again, haven't encountered any problems. Games that rely on things from the new client will still be broken in some manner.

Websearched packageinfo.vdf and found this: https://github.com/SteamDatabase/SteamAppInfo#packageinfovdf

They made a converter to do edits to the packageinfo.vdf allowing you to use the old Steam client.

The thread started here https://steamcommunity.com/groups/SteamClientBeta/discussions/3/1607148447829468911/?ctp=346

**Comment by Jonius7** [comment]
I have moved onto the new UI since mid June when they first had those package errors. But not without making CSS changes first. If you used the old list or grid view, it is hard to recreate that in the new client without some JS (which I still haven't found someone knowing how to do), but otherwise everything terrible about the new UI can be customised with CSS.

https://gyazo.com/aa37f6846e6771ecbc51f08808b93864
